action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Do Not Submit'
+description: 'Prevent pull requests from being submitted by failing if changes contain a key word'
+inputs:
+  keyword:
+    description: 'The keyword to search for in the file changes. Must not be empty.'
+    required: true
+    default: 'DO_NOT_SUBMIT'
+  allow_list:
+    description: 'A list of files to search for the keyword'
+    required: true
+    default: '*'
+  ignore_list:
+    description: 'A list of files to ignore when searching for the keyword'
+    required: false
+    default: ''
+outputs: {} # TODO??
+runs:
+  using: "composite"
+  steps:
+    - name: Fetch changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v37
+    - name: Check files for ${{ inputs.keyword }}
+      id: check-files
+      shell: bash
+      # TODO: Does `all_modified_files` include new, copied, and type_changed files?
```

Comment:
Does this help clarify? https://github.com/marketplace/actions/changed-files?version=v37#outputs

Reply:
FYI looks like all_modified_files was removed between v35 and v37

Reply:
Lol yeah. Your latest changes from `all_modified_files ` to `all_changed_files` is what clicked for me to update this 🍾